<?xml version="1.0"?>
<launch>
    <arg name="cfg_path"                 default="$(find uv_network)/config"/>
    <arg name="GPS_config"               default="$(arg cfg_path)/GPS.yaml"/>
    <!--<arg name="range_config"             default="$(arg cfg_path)/range.yaml"/>    -->

<group ns="external">
    <!-- lanza el nodo gps -->
    <node pkg="uv_network" name="gps" type="gps.py" output="screen">
        <param name="node_id" value="GPS"/>
        <rosparam command="load" file="$(arg GPS_config)"/>
        <param name="directory" value="$(find uv_network)/sim/"/>        
        <rosparam param="rate" subst_value="True">5.</rosparam>
        <rosparam param="topics" subst_value="True">
        {subs: {'1':0.5,'2':0.5}, pubs: ['gps_raw_data']}
        </rosparam>
        <rosparam param="fieldnames" subst_value="True">
        ['id','gps_stamp','gps_x','gps_y']
        </rosparam>
    </node> 
</group>

</launch>